spot-ml/install_ml.sh

```diff
@@ -1,12 +0,0 @@
- #!/bin/bash
```

Comment:
so now we have to manually copy the jar and resources and ops script to the MLNODE? why not have the installation script do those three copies for you?